docs/architecture/adr-074-timeout-params.md

```diff
@@ -66,8 +66,8 @@ The 8 timeout parameters will be consolidated down to 6. These will be as follow
 	 parameters.
 * `TimeoutCommit`
 	* Same as current `TimeoutCommit`.
-* `SkipTimeoutCommit`
-	* Same as current `SkipTimeoutCommit`.
+* `TimeoutCommitBypassEnabled`
```

Comment:
I agree with the reasoning here—but since all the other names here beginning with "Timeout" are, in fact, timeouts, what about maybe: `EnableTimeoutCommitBypass` to signal that this one is a flag?

Reply:
`EnableTimeoutCommitBypass` doesn't sound too different to `SkipTimeoutCommit`. Maybe I'm being too fussy but I think we shouldn't refer to `TimeoutCommit` in this flag as they are really triggered at two different thresholds. TimeoutCommit starts at 2/3, this bypass is triggered once we receive 100% of the votes. So they kind of have two different starting points. `EnableProgressionAfterAllVotes`? `EnableInstantProgression`. Maybe there's no right way to name this variable and we just need to be sure that the documentation is comprehensive

Reply:
I think that the logic is somewhat too complex to describe in shortish variable name.

Reply:
Okay, I only ask then that perhaps we don't use `TimeoutCommit` in the name of this variable. I think it's better if we don't try to associate these two variables together